jsk_2015_06_hrp_drc/drc_task_common/CMakeLists.txt

```diff
@@ -12,7 +12,10 @@ add_message_files(DIRECTORY msg FILES Float32ArrayStamped.msg)
 add_message_files(DIRECTORY msg FILES Int8Float64.msg)
 add_service_files(DIRECTORY srv FILES RvizMenuCall.srv RvizMenuSelect.srv EusCommand.srv StringRequest.srv ICPService.srv GetIKArm.srv GetIKArmPose.srv GoPosCommand.srv Uint8Request.srv AngleVectorCommand.srv)
 
-generate_messages(DEPENDENCIES ${PCL_MSGS} std_msgs std_srvs visualization_msgs sensor_msgs geometry_msgs jsk_pcl_ros jsk_interactive_marker jsk_recognition_msgs move_base_msgs)
+# Generate msg file from DRCParametersConfig file
+execute_process(
```

Comment:
@garaemon why you run gen_reconfigure_msg.py??
we cann't compile jsk_demos
https://travis-ci.org/jsk-ros-pkg/jsk_demos/jobs/114982208